test/Event/EventTest.php

```diff
@@ -125,6 +126,38 @@ public function it_throws_an_error_when_creating_an_event_with_a_non_string_even
         );
     }
 
+    /**
+     * @test
+     */
+    public function it_sets_the_audience_type_to_education_when_creating_an_event_with_a_dummy_education_location()
+    {
+        $locationUuid = UUID::generateAsString();
+        LocationId::setDummyPlaceForEducationIds([$locationUuid]);
+
+        $event = Event::create(
+            'd2b41f1d-598c-46af-a3a5-10e373faa6fe',
```

Comment:
I prefer using an `$eventId = UUID::generateAsString()` that can be reused on line 147, but definitely not a blocking issue :)